rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -1023,9 +1023,9 @@ void VKGSRender::on_init_thread()
 
 	if (!m_overlay_manager)
 	{
-		m_frame->hide();
+		show_window();
```

Comment:
This can't have been tested, this path is for Qt only, not native.

Reply:
it's actually wrong to change these lines.
we want the window to pop up after shaders were compiled.
so there's no use in adding the visibility workflow here

Reply:
Yea, that's what made me think it was a copypasta